csharp/test/Ice/metrics/AllTests.cs

```diff
@@ -612,7 +612,7 @@ public static IMetricsPrx allTests(TestHelper helper, CommunicatorObserver obsv)
                 metrics.Clone(connectionTimeout: 500).opByteS(new byte[10000000]);
```

Comment:
not clear why we do this clone if we are setting the same timeout, isn't this returning the same proxy?

Reply:
nvm the previous call didn't assign the proxy